pkg/network/protocols/kafka/testdata/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     tmpfs:
       - /bitnami/zookeeper/data
   kafka:
-    image: bitnami/kafka:3.4
+    image: bitnami/kafka:3.7
```

Comment:
you need to modify ami-builder repository

Reply:
and system-probe load tests
https://github.com/DataDog/system-probe-test-environments/blob/4d92a3613129020dba3eef55226578e917f06e32/k8s/kafka-load-test/templates/kafka.yaml#L38C6-L38C43